test/unit-test-helpers/mock_ima.js

```diff
@@ -64,7 +64,8 @@ google =
       setPlayerVersion : function() {},
       setPlayerType : function() {},
       setVpaidMode : function() {},
-      setLocale : function() {}
+      setLocale : function() {},
+      setDisableFlashAds : function() {},
```

Comment:
Added below.  I dont think I can run the actual ad request code, where we set the param in the ima sdk but I can test that loadmetadata is reading it in correctly.